src/backend/Csrs.Api/Instrumentation/InstrumentationExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+namespace Microsoft.Extensions.DependencyInjection;
```

Comment:
Use namespace so that in Program.cs, we do not require additional using statements and the `AddXXX` method is just available.